packages/server/src/auth/api/member.ts

```diff
@@ -0,0 +1,102 @@
+import { Member } from '@prisma/client'
+import { booleanArg, intArg, mutationField, nonNull, objectType, queryField, stringArg } from 'nexus'
+import { Member as NexusMember } from 'nexus-prisma'
```

Comment:
Here it's definitely because the Prisma client wasn't generated because `nexus-prisma` is one of the 2 providers so it's types will be wrong until `prisma generate` is ran successfully.